api/src/main/java/com/strategyobject/substrateclient/api/pallet/parachain_system/ParachainSystem.java

```diff
@@ -0,0 +1,81 @@
+package com.strategyobject.substrateclient.api.pallet.parachain_system;
+
+import com.strategyobject.substrateclient.pallet.annotation.*;
+import com.strategyobject.substrateclient.rpc.api.primitives.Hash;
+import com.strategyobject.substrateclient.scale.ScaleType;
+import com.strategyobject.substrateclient.scale.annotation.Scale;
+import com.strategyobject.substrateclient.scale.annotation.ScaleReader;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigInteger;
+
+
+@Pallet("ParachainSystem")
+public interface ParachainSystem {
+    /**
+     * The validation function has been scheduled to apply.
+     */
+    @Event(index = 0)
+    @Getter
```

Comment:
I think we can remove the Getter and Setter annotations on events that don't have any parameters